apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -5,7 +5,7 @@ import ProgressButton from "../progress/ProgressButton";
 
 const styles = {
   section: {
-    width: 970,
+    width: 960,
```

Comment:
I made this component less wide because the thick dashed border is on the outside of the component and this width aligns it better with other page elements.